Application/components/TextChannels/TextChannelItem.tsx

```diff
@@ -0,0 +1,33 @@
+import cx from 'clsx';
+import { rem, Text, Stack } from '@mantine/core';
+import { IconGripVertical } from '@tabler/icons-react';
+import { useListState } from '@mantine/hooks';
+import { DragDropContext, Droppable, Draggable } from '@hello-pangea/dnd';
+import { TextChannelItemProps } from '@/accordTypes';
+import classes from './TextChannelItem.module.css';
+
+export const TextChannelItem: React.FC<TextChannelItemProps> = ({ id, index, channelName, numberOfMembers, onClick }) => {
+  const symbol = channelName.substring(0, 2).toUpperCase(); // Generate symbol from channelName
```

Comment:
- Implement error handling for edge cases such as when the channelName is null or empty. Handle such scenarios gracefully to prevent runtime errors and ensure that the component behaves predictably regardless of the input data.